rebase_test.go

```diff
@@ -0,0 +1,402 @@
+package git
+
+import (
+	"errors"
+	"strconv"
+	"testing"
+	"time"
+)
+
+// Tests
+
+func TestDefaultRebaseOptions(t *testing.T) {
+	opts, err := DefaultRebaseOptions()
+	checkFatal(t, err)
+
+	if opts.Version != 1 {
+		t.Error("Expected opts Version to equal 1, got ", opts.Version)
+	}
```

Comment:
The version it needs to return in the version that is specified by the libgit2 headers in the `#define`. That's what the compiler is building and that's what it will use in order to figure out the size of the struct, which has to match what we're telling libgit2 we have.